src/components/date-input/_date-input.scss

```diff
@@ -10,18 +10,17 @@
 @include govuk-exports("govuk/component/date-input") {
   .govuk-date-input {
     @include govuk-clearfix;
+    font-size: 0;
```

Comment:
I think this could do with a comment explaining why it's needed / what it's doing.